src/repr/Cargo.toml

```diff
@@ -44,6 +44,7 @@ uuid = "0.8.2"
 [dev-dependencies]
 criterion = { git = "https://github.com/MaterializeInc/criterion.rs.git" }
 rand = "0.8.5"
+proptest-derive = { git = "https://github.com/materializeInc/proptest.git" }
```

Comment:
You might wanna cherry-pick e19a146c27f171e4d5b258178736051b3e3eedbc (currently in #11728) in this PR (I will remove it from #11728 if this gets merged first).